config/routes.rb

```diff
@@ -18,4 +18,12 @@
       resources :likes, only: [:new, :create]
     end
   end
+
```

Comment:
- [ ] Kindly in order to review the routes, first fix the command to create the database and load the seeds. There are some problems with your schema as I can see. So this will difficult your creating post and comments

![image](https://github.com/Habtamu210/Blog-App/assets/93630700/859329be-142a-4249-9b81-158706bd9a87)